src/index.js

```diff
@@ -135,9 +135,22 @@ const getPreProcessorsConfig = (function wrap() {
     };
 }());
 
+const getGlobalSassData = (rootDir) => {
+    let sassrc;
+
+    try {
+        sassrc = require(resolve(rootDir, '.sassrc.js'));
+    } catch (e) {
+        sassrc = { data: '' };
+    }
+    
+    return sassrc.data || '';
+};
+
 module.exports = {
     process(src, path, config) {
         const preProcessorsConfig = getPreProcessorsConfig(config.rootDir);
+        const globalSassData = getGlobalSassData(config.rootDir);
```

Comment:
It should be lazy `require` inside `switch` to avoid needless operation for not sass users.